tests/figures/chart/chart_plugin.test.ts

```diff
@@ -1492,6 +1492,181 @@ describe("title", function () {
       expect(scales.x!["title"].font.style).toEqual("italic");
     }
   );
+
+  test("line chart runtime reflects axis bounds, grids and scale type", () => {
+    setGrid(model, {
+      A1: "Month",
+      A2: "Jan",
+      A3: "Feb",
+      A4: "Mar",
+      B1: "Series A",
+      B2: "5",
+      B3: "15",
+      B4: "25",
+    });
```

Comment:
nitpick: dataset is kinda useless for the test no ? Just one random point/label should be fine. Same in the otehr tests